integration-cli/docker_cli_daemon_test.go

```diff
@@ -2190,3 +2190,17 @@ func (s *DockerSuite) TestDaemonDiscoveryBackendConfigReload(c *check.C) {
 	c.Assert(out, checker.Contains, fmt.Sprintf("Cluster Store: consul://consuladdr:consulport/some/path"))
 	c.Assert(out, checker.Contains, fmt.Sprintf("Cluster Advertise: 192.168.56.100:0"))
 }
+
+// Test for #21956
+func (s *DockerDaemonSuite) TestDaemonInspectSyslog(c *check.C) {
```

Comment:
Thanks @vdemeester @cpuguy83 I changed the test name to `TestDaemonLogOptions`. Don't know if this one looks better but let me know if there are better names that could fit this test.